### PR TITLE
switch from Ffaker to Faker

### PR DESCRIPTION
**Pull request by jmkoni**
Per issue #171, we are interested in replacing Ffaker with Faker since it is more robust at this point.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,8 +13,9 @@ gem 'chosen-rails'
 gem 'devise', '~> 4.2'
 gem 'devise_invitable', '~> 1.7.0'
 gem 'devise_security_extension', git: 'https://github.com/phatworx/devise_security_extension.git'
-gem 'ffaker', require: false
+gem 'faker', require: false
```
Comment (latazzajones) [inline Gemfile]: 👏 YAS!

File: Gemfile [hunk 2]
```diff
@@ -13,8 +13,9 @@ gem 'chosen-rails'
...
 gem 'filterrific', '~> 5.x'
+gem 'geography_helper'
```
Comment (latazzajones) [inline Gemfile]: +1 for alphabitizing ❤️

File: Gemfile.lock [hunk 3]
```diff
@@ -163,25 +166,25 @@ GEM
...
-    mime-types-data (3.2016.0521)
+    mime-types-data (3.2018.0812)
     mini_mime (1.0.1)
     mini_portile2 (2.3.0)
     minitest (5.11.3)
     multi_json (1.11.3)
-    mustermann (1.0.2)
-    newrelic_rpm (3.17.1.326)
+    mustermann (1.0.3)
+    newrelic_rpm (5.7.0.350)
```
Comment (latazzajones) [inline Gemfile.lock]: This is a really big jump. I like the idea of updating all of the gems frequently but would it be worth only installing Faker and removing FFaker. By doing a general gem update in a seperate pr as a practice we could make it easier to address breaking changes from our dependancies. I can go either way. It looks like the only things that made version jumps here are non critical. 

Thoughts?

File: Gemfile.lock [hunk 4]
```diff
@@ -295,21 +298,21 @@ GEM
...
       rack (~> 2.0)
-      rack-protection (= 2.0.3)
+      rack-protection (= 2.0.5)
       tilt (~> 2.0)
-    skylight (2.0.2)
-      skylight-core (= 2.0.2)
-    skylight-core (2.0.2)
+    skylight (3.1.4)
+      skylight-core (= 3.1.4)
+    skylight-core (3.1.4)
```
Comment (latazzajones) [inline Gemfile.lock]: another version jump here.

File: Gemfile.lock (hunk 3 above)
Comment (jmkoni) [reply Gemfile.lock]: Yeah I could undo the changes and put up a separate PR. That might be easier in case something causes an error

**Comment by jmkoni** [comment]
@latazzajones not sure why they all updated initially. but got it straightened out 😅

**Comment by CZagrobelny** [comment]
thanks @jmkoni and @latazzajones!  Should have time to do a final review in the next couple days!

**Comment by jmkoni** [comment]
After this, maybe we should consider doing an upgrade? Might be worth upgrading our version of ruby as well so we don't have to do a major upgrade later. Thoughts?

**Comment by latazzajones** [comment]
@jmkoni I think that's a great idea - would you mind opening an issue for it and we can have convo about it there? Then feel free to grab the issue if it's something you're willing to pick up!

**Pull request merged**